### ISSUE TITLE
log-average missing rate

### ISSUE THREAD

**Issue by Ilbotre** [Cartucho/mAP#112]
Do you have provided code in order to find out the log-average missing rate evaluation metric too?

**Comment by Cartucho** [comment]
Yeah, it is there. But I'm not sure if it is correct. Currently, I do not have any sample to test it.

**Comment by Ilbotre** [comment]
thank you for your answer.
one more thing: In the "ground-truth"  and "detection-results" files the <left> <top> <right> <bottom> parameter **have to be integer** or they can be float too? in particular for the "detection-results" file.
Thank you very much
